signac/contrib/import_export.py

```diff
@@ -293,25 +333,18 @@ def _export_jobs(jobs, path, copytree):
     dst : str
         Destination path.
 
-    Raises
-    ------
-    RuntimeError
-        If paths generated with given path function are not unique.
```

Comment:
This can still raise a RuntimeError, so we shouldn't delete this. (The errors listed in a function's docstring may be raised by the code it calls, like in this case with `_check_path_function`.)